lib/pf/locationlog.pm

```diff
@@ -185,7 +186,7 @@ sub locationlog_db_prepare {
             LEFT JOIN node USING (mac)
         WHERE switch=? AND port=?
             AND (node.voip='' OR node.voip='no')
-            AND (ISNULL(end_time) OR end_time = 0)
+            AND end_time = 0
```

Comment:
The bigger questions is do we keep a space between the operators and operands in sql statements.

Because if I am going to take the time to change "end_time = 0" to "end_time=0".
I would also take the time to change everywhere else (in the module) to be consistent.